web/yo/app/views/gene.html

```diff
@@ -540,6 +540,11 @@ <h4 ng-if="!tumor.trials_review.get('action') && !tumor.trials_review.get('remov
       </div>
     </div>
   </div>
+    <div style="display: inline-block">
+        <br/><br/>
```

Comment:
Try to use css style instead of line break in here. `<br/>` is supposed to indicate a new line, not for separating sections.